src/vs/workbench/electron-browser/actions/developerActions.ts

```diff
@@ -173,6 +173,7 @@ export class ToggleScreencastModeAction extends Action {
 		keyboardMarker.style.textAlign = 'center';
 		keyboardMarker.style.fontSize = '56px';
 		keyboardMarker.style.display = 'none';
+		keyboardMarker.style.overflow = 'hidden';
```

Comment:
Nice solution.
What do you think about a pure css solution, just using ellipsis at starting, something like:
```typescript
keyboardMarker.style.whiteSpace = 'nowrap';
keyboardMarker.style.overflow = 'hidden';
keyboardMarker.style.textOverflow = 'ellipsis';
keyboardMarker.style.direction = 'rtl';

Reply:
Yeah that looks much better, without the while loop truncating the text it could in theory cause memory issues but I think that's minor compared to the simpler code